test_script.sh

```diff
@@ -3,5 +3,8 @@ cmake ..
 cmake --build .
 cd tests
 ctest --rerun-failed --output-on-failure
-cd ../src
-python -m pytest ../../tests/python
\ No newline at end of file
+cd ../../python/pythonmonkey
+ln -sf ../../build/src/pythonmonkey.so .
+cd ../../tests/python
+ln -sf ../../python/pythonmonkey .
+python -m pytest .
```

Comment:
Just a reminder that if you run `poetry install` in the project root, it will build and install a link in the poetry python env allowing you to import from `python/pythonmonkey` without needing to put it in a particular location so long as you run commands with poetry (i.e. `poetry run pytest ./tests/python `)


Using poetry to unify many different methods of doing things should aid us in keeping code consistent. Let me know if you'd like some more advice on this. 

edit: For additional clarity: `poetry run python -c "import pythonmonkey;print(pythonmonkey.__path__)"` will result in `$PROJECT_ROOT/python/pythonmonkey`

Reply:
Do you think it's a good idea to update the entire `Build Instructions` and `Running tests` in readme to use `poetry`?

Reply:
Yeah I think so. I think it'll help us make sure all devs follow similar processes and we can mitigate the number of different ways people build/run tests and unify them into one, simplified, process.